src/cli.ts

```diff
@@ -479,4 +492,30 @@ async function withdraw_getHash(ctx: Context, to: string, amount: number, id: st
 async function withdraw_useSignature(ctx: Context, id: string) {
   const txId = await sendSignedWithdrawalTransaction(ctx, id)
   logSuccess(`Transaction with id ${txId} sent to the node`)
-}
\ No newline at end of file
+}
+
+
+//////////////////////////////////////////////////////////////////////////////////////////
+// Transaction signing
+
+export async function signAndSend(network: string, envPath: string, id: string) {
+  const ctx: Context = contextEnv(envPath, network)
+  const path = `${forDefiDirectory}/${forDefiUnsignedTxnDirectory}/${id}.unsignedTx.json`
+  const json = fs.readFileSync(path, 'utf8')
+  const tx = JSON.parse(json)
+
+  if (!tx) throw new Error("Invalid txn file")
+  const response = await getUserInput(`${colorCodes.redColor}Warning: You are about to expose your private key to 800+ dependencies, and we cannot guarantee one of them is not malicious! \nThis command is not meant to be used in production, but for testing only!${colorCodes.resetColor} \nProceed? (Y/N) `)
+  if (response == 'Y' || response == 'y') {

```

Comment:
instead of response == Y || response == "y" convert to lower or uppercase and check the condition